### ISSUE TITLE
During an edits generation, a scrollbar appears below the tab being edited

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#236110]
This only happened once so far, but I have tab wrapping enabled and a good amount of tabs open

![Image](https://github.com/user-attachments/assets/54264444-5723-482c-836f-5eb025faff43)
<img width="1262" alt="Image" src="https://github.com/user-attachments/assets/c519edef-ed37-4f18-b415-693cc2d666bb" />

**Comment by jrieken** [comment]
@sbatten I don't believe this is related to /edits but please add actionable information

**Comment by sbatten** [comment]
I was able to reproduce this again, I believe these steps should be reliable.

- Enable `workbench.editor.wrapTabs`
- Ensure that you have enough tabs open so that wrapping occurs
- Use copilot edits to generate an edit on a file in the top row of tabs e.g. "Untitled-4" in the attached image.

This should render the scrollbar during progress of the edit.

**Comment by jrieken** [comment]
Looks like an issue with tabs, while generating code we do add decoration (for progress and then for having changes)
